src/umtool/ipac4_2015_default.json

```diff
@@ -2,7 +2,7 @@
   "version" : 2,
   "product" : "ipac4",
   "config" : {
-    "debounce" : "standard"
+    "debounce" : "standard",
```

Comment:
Don't think this coma is allowed, since it is the last entry.

Reply:
No it's not,
I initially had the paclink config, removed it but forgot to remove the comma.
Strange the comma is removed on my branch..... but didn't pull across....